src/tests-environments/meteor.ts

```diff
@@ -64,6 +66,46 @@ export class MeteorStressTest {
       this.subscriptions[subscriptionName] = result;
     });
   }
+  
+  call(methodName: string, ...args): Promise<any> {
+    return new Promise((resolve, reject) => {
+      this.ddpClient.call(methodName, args, (err, result) => {
+        if (err) {
+          reject(err);
+        }
+        else {
+          resolve(result);
+        }
+      });
+    });
+  }
+  
+  login(username: string, email: string, password: string): Promise<any> {
```

Comment:
Maybe it worth changing `Promise<any>` to `Promise<Meteor.User>` (is that the actual value at all?) to make it easier to use